.github/workflows/ee-nlc-snapshot-push.yml

```diff
@@ -13,8 +13,16 @@ env:
   test_container_name_ee: hazelcast-ee-test
 
 jobs:
+  jdks:
```

Comment:
We've backported https://github.com/hazelcast/hazelcast-docker/pull/715 as well?

Reply:
Yes, you're right, it was easier and safer then fighting with more conflicts